system-tests/lib/cre/contracts/data_feeds.go

```diff
@@ -0,0 +1,42 @@
+package contracts
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	libc "github.com/smartcontractkit/chainlink/system-tests/lib/conversions"
+)
+
+// GetDecimalsFromFeedID extracts the number of decimals from a feed ID
+// Feed ID format is assumed to be a hex string (with or without 0x prefix)
+func GetDecimalsFromFeedID(feedID string) (int, error) {
```

Comment:
Shouldn't deployment for DF or Keystone provide us with this function? What if encoding will change? @Tofel @karen-stepanyan

Reply:
That makes sense. I can add a similar function to our `changeset` package.